frontend/components/common/Layout/index.tsx

```diff
@@ -306,7 +312,13 @@ function LayoutNavigationDrawer({ menu }: LayoutNavigationDrawerProps) {
          <DrawerCloseButton />
          <NextLink href="/" passHref>
             <WordmarkLink aria-label="Go to homepage" mb="8">
-               <Wordmark />
```

Comment:
Should we delete the `Wordmark` component, since it's no longer used? I guess we'll probably bring it back later

Reply:
Thats my thinking is we will switch back at some point